python/setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name='Golem-Task-Api',
-    version='0.15.0',
+    version='0.16.0',
```

Comment:
it looks like wait_for_socket_open is now private to task_api so there is no public API change
Maybe we can add a `_` to the function name, it used to be public